drivers/sensor/amd_sb_tsi/sb_tsi.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_SB_TSI_SB_TSI_H_
+#define ZEPHYR_DRIVERS_SENSOR_SB_TSI_SB_TSI_H_
+
+#include <zephyr/device.h>
+#include <zephyr/sys/util.h>
+
+#define SB_TSI_TEMP_INT 0x01
+#define SB_TSI_TEMP_DEC 0x10
+#define SB_TSI_TEMP_DEC_SHIFT 5
+#define SB_TSI_TEMP_DEC_SCALE 8
+
+struct sb_tsi_data {
+	uint8_t sample_int;
+	uint8_t sample_dec;
+};
+
+struct sb_tsi_config {
+	struct i2c_dt_spec i2c;
+};
```

Comment:
There's not enough here to justify a separate private header file; please move these definitions to the .c file